DashAISoftware/DashAI
# Prepare Dataset View

# Summary


Adds Prepare dataset step in the New Experiment flow

## Type of change



- Front end new feature.

## Changes


- Insert dataset columns that you wanna use as input for the model.
- Insert dataset columns that you wanna use as output or class label for the model.
- Insert splits using random rows or specific for each split between training, validation and testing.
## How to Test


1. Go to ```DashAI/DashAI/front``` and run ```yarn build```
2. Go back to ```DashAI/``` and run ```python -c "import DashAI;DashAI.run()"```
3. Check the view in New Experiment.

## Screenshots


![image](https://github.com/DashAISoftware/DashAI/assets/70553082/3352b04e-9834-4120-9cd4-f7c9ab7646e4)
## Notes


- There's still things that needs to be added to this new feature such as validate the rows types with the task, evade overlaping between column and rows splits, etc.
